src/connector/src/sink/remote.rs

```diff
@@ -165,6 +165,11 @@ impl<R: RemoteSinkTrait> Sink for RemoteSink<R> {
 }
 
 async fn validate_remote_sink(param: &SinkParam, sink_name: &str) -> ConnectorResult<()> {
+    if sink_name == OpenSearchSink::SINK_NAME {
```

Comment:
it seems the other three sinks validate tier in `validate` while "opensearch" validate in `validate_remote_sink`, is it intended?